ios/Podfile.lock

```diff
@@ -795,7 +795,7 @@ SPEC CHECKSUMS:
   DoubleConversion: cf9b38bf0b2d048436d9a82ad2abe1404f11e7de
   EXHaptics: 337c160c148baa6f0e7166249f368965906e346b
   FBLazyVector: 7b423f9e248eae65987838148c36eec1dbfe0b53
-  FBReactNativeSpec: e564123bce1111e84dc7aa0765fb1175f0c48aa0
+  FBReactNativeSpec: d65967936e86fe0fe6cca5c0125c237636868d4a
```

Comment:
Unrelated to the real problem here, but this should probably be updated according to this: https://expensify.slack.com/archives/C01GTK53T8Q/p1626728088492500

To fully confirm, can you also run `pod install` in the `ios` folder on the `main` branch and verify that you also get a different checksum? Also, run it on this branch, `amal-participants-full-name-check`, and it should give you the same checksum as me.

Reply:
I got a different checksum for a bunch of things

RNReanimated (2.3.0-alpha.1):
DoubleConversion: cde416483dac037923206447da6e1454df403714
FBReactNativeSpec: 76cede005e0667703f69105017804cc6c5e39fc8
glog: 40a13f7840415b9a77023fbcae0f1e6f43192af3
RNReanimated: 833ebd229b31e18a8933ebd0cd744a0f47d88c42

Reply:
Ah hmm let me pull main into this branch and see if that resolves it. If not I might just not mess with this part.